lib/galaxy/datatypes/blast.py

```diff
@@ -0,0 +1,281 @@
+"""NCBI BLAST datatypes.
+
+Covers the ``blastxml`` format and the BLAST databases.
+"""
+
+import logging
+import os
+from time import sleep
+
+from galaxy.datatypes.data import get_file_peek
+from galaxy.datatypes.data import Data, Text
+from galaxy.datatypes.xml import GenericXml
+
+log = logging.getLogger(__name__)
+
+
+class BlastXml(GenericXml):
+    """NCBI Blast XML Output data"""
```

Comment:
I guess we should annotate them with the EDAM ontology as the other datatypes: https://bioportal.bioontology.org/ontologies/EDAM?p=classes&conceptid=http%3A%2F%2Fedamontology.org%2Fformat_1333

@odoppelt can you give us a hand on this?